packages/bot-web-ui/src/components/load-modal/load-modal.scss

```diff
@@ -1,21 +1,51 @@
 #modal_root {
     .load-strategy {
-        &__recent-item {
-            grid-template-columns: 1fr 1fr;
-        }
+        &__recent {
+            &-item {
+                padding: 1.6rem;
+                grid-template-columns: 1fr 1fr;
+                &-location {
+                    justify-content: center;
+                }
+                &-text {
+                    height: 10rem;
+                    flex-direction: column;
+                    text-align: left;
+                }
+                &-title {
+                    font-size: var(--text-size-s);
+                }
+                &-saved {
```

Comment:
@rupato-deriv related to methodology of BEM it should be with syntax Modifier: `'&--saved'`.
Instead of` '&-text'` should be Element as `'&__text'`, instead of` '&-title'` should  be `'&__title'` and so on..
 Please check this article https://getbem.com/naming/

Reply:
Please check all of your code of the styles on this way

Reply:
@maryiafrantsava-binary this is a legacy to change  the css here it needs some time to be refactored.